apps/emojilingo/emojilingo.go

```diff
@@ -0,0 +1,25 @@
+// Package emojilingo provides details for the Emoji Lingo applet.
+package emojilingo
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed emoji_lingo.star
+var source []byte
+
+// New creates a new instance of the Emoji Lingo applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "emoji-lingo",
+		Name:        "Emoji Lingo",
+		Author:      "Cedric Sam",
+		Summary:     "A random emoji, localized",
+		Desc:        "Displays a random emoji and its unique short text annotation from the Unicode Consortium in a given locale.",
```

Comment:
Many non-technical users will likely not understand what "locale" means in this context.